central/notifiers/cscc/cscc_test.go

```diff
@@ -0,0 +1,87 @@
+package cscc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/golang/mock/gomock"
+	clusterMocks "github.com/stackrox/rox/central/cluster/datastore/mocks"
+	"github.com/stackrox/rox/central/notifiers/cscc/findings"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWithFakeCSCC(t *testing.T) {
+
+	const (
```

Comment:
[nit] Either just make it a var and keep it in the func or move the const outside of the function